cat: .experiment_zone: No such file or directory
Request issued for: [node-2]
done: true
metadata:
  '@type': type.googleapis.com/google.cloud.common.OperationMetadata
  apiVersion: v1
  cancelRequested: false
  createTime: '2021-04-06T23:13:41.077533063Z'
  endTime: '2021-04-06T23:14:13.731100956Z'
  target: projects/red-atlas-305317/locations/europe-west4-a/nodes/node-2
  verb: update
name: projects/red-atlas-305317/locations/europe-west4-a/operations/operation-1617750821045-5bf55f70605fb-646ca22d-5961b6e5
response:
  '@type': type.googleapis.com/google.cloud.tpu.v1.Node
  acceleratorType: v3-8
  apiVersion: V1
  cidrBlock: 10.73.142.64/29
  createTime: '2021-03-13T00:10:24.868252445Z'
  health: HEALTHY
  ipAddress: 10.73.142.66
  name: projects/red-atlas-305317/locations/europe-west4-a/nodes/node-2
  network: global/networks/default
  networkEndpoints:
  - ipAddress: 10.73.142.66
    port: 8470
  port: '8470'
  schedulingConfig: {}
  serviceAccount: service-276208758346@cloud-tpu.iam.gserviceaccount.com
  state: READY
  symptoms:
  - createTime: '2021-03-25T19:28:21.704126002Z'
    details: The TPU HBM has run OOM at timestamp 2021-03-25T19:28:19Z
    symptomType: HBM_OUT_OF_MEMORY
    workerId: '0'
  - createTime: '2021-03-28T21:10:24.483577115Z'
    details: The TPU HBM has run OOM at timestamp 2021-03-28T21:10:19Z
    symptomType: HBM_OUT_OF_MEMORY
    workerId: '0'
  tensorflowVersion: 2.4.1
Waiting for operation [projects/red-atlas-305317/locations/europe-west4-a/operations/operation-1617750821045-5bf55f70605fb-646ca22d-5961b6e5] to complete...
.............................................................................................................................................................................................done.
europe-west4-a
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/cam2tensorflow/TensorFlowModels/yolo/train_vm.py", line 157, in <module>
    app.run(main)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 300, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "/home/cam2tensorflow/TensorFlowModels/yolo/train_vm.py", line 115, in main
    params = train_utils.parse_configuration(FLAGS)
  File "/home/cam2tensorflow/TensorFlowModels/official/core/train_utils.py", line 249, in parse_configuration
    params, config_file, is_strict=True)
  File "/home/cam2tensorflow/TensorFlowModels/official/modeling/hyperparams/params_dict.py", line 441, in override_params_dict
    params.override(yaml.load(f, Loader=yaml.FullLoader), is_strict)
  File "/usr/local/lib/python3.7/dist-packages/yaml/__init__.py", line 112, in load
    loader = Loader(stream)
  File "/usr/local/lib/python3.7/dist-packages/yaml/loader.py", line 24, in __init__
    Reader.__init__(self, stream)
  File "/usr/local/lib/python3.7/dist-packages/yaml/reader.py", line 85, in __init__
    self.determine_encoding()
  File "/usr/local/lib/python3.7/dist-packages/yaml/reader.py", line 124, in determine_encoding
    self.update_raw()
  File "/usr/local/lib/python3.7/dist-packages/yaml/reader.py", line 178, in update_raw
    data = self.stream.read(size)
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/lib/io/file_io.py", line 117, in read
    self._preread_check()
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/lib/io/file_io.py", line 80, in _preread_check
    compat.path_to_str(self.__name), 1024 * 512)
tensorflow.python.framework.errors_impl.NotFoundError: yolo/configs/experiments/full_cfg_exp/yolov4-coco-adam-mo2017-720.yaml; No such file or directory
yolo_custom
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   226    0     2  100   224     11   1333 --:--:-- --:--:-- --:--:--  1345
okRequest issued for: [node-2]
done: true
metadata:
  '@type': type.googleapis.com/google.cloud.common.OperationMetadata
  apiVersion: v1
  cancelRequested: false
  createTime: '2021-04-06T23:14:23.932950702Z'
  endTime: '2021-04-06T23:17:07.191132843Z'
  target: projects/red-atlas-305317/locations/europe-west4-a/nodes/node-2
  verb: update
name: projects/red-atlas-305317/locations/europe-west4-a/operations/operation-1617750863914-5bf55f994262c-4df7a01f-52db843c
response:
  '@type': type.googleapis.com/google.cloud.tpu.v1.Node
  acceleratorType: v3-8
  apiVersion: V1
  cidrBlock: 10.73.142.64/29
  createTime: '2021-03-13T00:10:24.868252445Z'
  health: HEALTHY
  ipAddress: 10.73.142.66
  name: projects/red-atlas-305317/locations/europe-west4-a/nodes/node-2
  network: global/networks/default
  networkEndpoints:
  - ipAddress: 10.73.142.66
    port: 8470
  port: '8470'
  schedulingConfig: {}
  serviceAccount: service-276208758346@cloud-tpu.iam.gserviceaccount.com
  state: STOPPED
  symptoms:
  - createTime: '2021-03-25T19:28:21.704126002Z'
    details: The TPU HBM has run OOM at timestamp 2021-03-25T19:28:19Z
    symptomType: HBM_OUT_OF_MEMORY
    workerId: '0'
  - createTime: '2021-03-28T21:10:24.483577115Z'
    details: The TPU HBM has run OOM at timestamp 2021-03-28T21:10:19Z
    symptomType: HBM_OUT_OF_MEMORY
    workerId: '0'
  tensorflowVersion: 2.4.1
Waiting for operation [projects/red-atlas-305317/locations/europe-west4-a/operations/operation-1617750863914-5bf55f994262c-4df7a01f-52db843c] to complete...
...............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................done.
cat: .experiment_zone: No such file or directory
Request issued for: [node-2]
Waiting for operation [projects/red-atlas-305317/locations/europe-west4-a/operations/operation-1617760792662-5bf584960d0fd-63bc3846-81e1ac36] to complete...
.............................aborted by ctrl-c.
ERROR: (gcloud.compute.tpus.start) Aborting wait for operation https://tpu.googleapis.com/v1alpha1/projects/red-atlas-305317/locations/europe-west4-a/operations/operation-1617760792662-5bf584960d0fd-63bc3846-81e1ac36.

europe-west4-a
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/cam2tensorflow/TensorFlowModels/yolo/train_vm.py", line 157, in <module>
    app.run(main)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 300, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "/home/cam2tensorflow/TensorFlowModels/yolo/train_vm.py", line 115, in main
    params = train_utils.parse_configuration(FLAGS)
  File "/home/cam2tensorflow/TensorFlowModels/official/core/train_utils.py", line 249, in parse_configuration
    params, config_file, is_strict=True)
  File "/home/cam2tensorflow/TensorFlowModels/official/modeling/hyperparams/params_dict.py", line 441, in override_params_dict
    params.override(yaml.load(f, Loader=yaml.FullLoader), is_strict)
  File "/usr/local/lib/python3.7/dist-packages/yaml/__init__.py", line 112, in load
    loader = Loader(stream)
  File "/usr/local/lib/python3.7/dist-packages/yaml/loader.py", line 24, in __init__
    Reader.__init__(self, stream)
  File "/usr/local/lib/python3.7/dist-packages/yaml/reader.py", line 85, in __init__
    self.determine_encoding()
  File "/usr/local/lib/python3.7/dist-packages/yaml/reader.py", line 124, in determine_encoding
    self.update_raw()
  File "/usr/local/lib/python3.7/dist-packages/yaml/reader.py", line 178, in update_raw
    data = self.stream.read(size)
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/lib/io/file_io.py", line 117, in read
    self._preread_check()
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/lib/io/file_io.py", line 80, in _preread_check
    compat.path_to_str(self.__name), 1024 * 512)
tensorflow.python.framework.errors_impl.NotFoundError: yolo/configs/experiments/full_cfg_exp/yolov4-coco-adam-mo2017-720.yaml; No such file or directory
yolo_custom
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   226    0     2  100   224     11   1302 --:--:-- --:--:-- --:--:--  1313
okRequest issued for: [node-2]
/home/cam2tensorflow/.experiment_start.sh: line 11:  6232 Killed                  TPU_NAME=$1 python3 -m $2.train_vm --mode=train_and_eval --experiment=$3 --config_file=$2/configs/experiments/$4.yaml --model_dir=gs://tensorflow2/$4
cat: .experiment_zone: No such file or directory
ERROR: (gcloud.compute.tpus.start) INVALID_ARGUMENT: StartNode is not supported when state is STOPPING
europe-west4-a
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/cam2tensorflow/TensorFlowModels/yolo/train_vm.py", line 157, in <module>
    app.run(main)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 300, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "/home/cam2tensorflow/TensorFlowModels/yolo/train_vm.py", line 115, in main
    params = train_utils.parse_configuration(FLAGS)
  File "/home/cam2tensorflow/TensorFlowModels/official/core/train_utils.py", line 249, in parse_configuration
    params, config_file, is_strict=True)
  File "/home/cam2tensorflow/TensorFlowModels/official/modeling/hyperparams/params_dict.py", line 441, in override_params_dict
    params.override(yaml.load(f, Loader=yaml.FullLoader), is_strict)
  File "/usr/local/lib/python3.7/dist-packages/yaml/__init__.py", line 112, in load
    loader = Loader(stream)
  File "/usr/local/lib/python3.7/dist-packages/yaml/loader.py", line 24, in __init__
    Reader.__init__(self, stream)
  File "/usr/local/lib/python3.7/dist-packages/yaml/reader.py", line 85, in __init__
    self.determine_encoding()
  File "/usr/local/lib/python3.7/dist-packages/yaml/reader.py", line 124, in determine_encoding
    self.update_raw()
  File "/usr/local/lib/python3.7/dist-packages/yaml/reader.py", line 178, in update_raw
    data = self.stream.read(size)
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/lib/io/file_io.py", line 117, in read
    self._preread_check()
  File "/usr/local/lib/python3.7/dist-packages/tensorflow/python/lib/io/file_io.py", line 80, in _preread_check
    compat.path_to_str(self.__name), 1024 * 512)
tensorflow.python.framework.errors_impl.NotFoundError: yolo/configs/experiments/full_cfg_exp/yolov4-coco-adam-mo2017-720.yaml; No such file or directory
yolo_custom
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   226    0     2  100   224     11   1309 --:--:-- --:--:-- --:--:--  1321
okRequest issued for: [node-2]
Waiting for operation [projects/red-atlas-305317/locations/europe-west4-a/operations/operation-1617760860863-5bf584d717b74-c1e3da4c-e959dd69] to complete...
...........................aborted by ctrl-c.
ERROR: (gcloud.compute.tpus.stop) Aborting wait for operation https://tpu.googleapis.com/v1alpha1/projects/red-atlas-305317/locations/europe-west4-a/operations/operation-1617760860863-5bf584d717b74-c1e3da4c-e959dd69.

